src/main/java/slimeknights/tconstruct/library/modifiers/Modifier.java

```diff
@@ -324,6 +330,41 @@ public void onInventoryTick(IModifierToolStack tool, int level, World world, Liv
   }
 
 
+  /**
+    * Called when this item is used when targeting a Block
+   * @param tool 
+   * @param level 
+   * @param stack 
+   * @param context        Full item use context
+   * @return  Action result
+   */
+  public ActionResultType onItemUse(ToolStack tool, int level, ItemStack stack, ItemUseContext context) {
+    return ActionResultType.PASS;
+  }
+  
+  public ActionResultType itemInteractionForEntity(ToolStack tool, int level, ItemStack stack, PlayerEntity playerIn, LivingEntity target, Hand hand) {
+    return ActionResultType.PASS;
+  }
+  
+  public ActionResult<ItemStack> onItemRightClick(IModifierToolStack tool, int level, World worldIn, PlayerEntity playerIn, Hand handIn) {
+    return ActionResult.resultPass(playerIn.getHeldItem(handIn));
+  }
+
+  /**
+  * @return  Whether the modifier should block any incoming ones from firing
+  */
+  public boolean onPlayerStoppedUsing(IModifierToolStack tool, int level, World worldIn, LivingEntity entityLiving, int timeLeft) {
```

Comment:
Might be worth mentioning how we get this hook to call in the javadoc (setting active hand, duration, etc.). I know its vanilla, but its useful information.

Reply:
That would mean i would need to learn all that. :'D

Reply:
IIRC, you use what the bow does, `setActiveHand` IIRC.